### ISSUE TITLE
Replace on_audio_query with flutter_audio_metadata or metadata_god

### ISSUE THREAD

**Issue by FriederHannenheim** [moritz-weber/mucke#99]
**Is your improvement request related to a problem? Please describe.**
As described in #70, flac dates currently don't show up correctly. This is due to a bug in on_audio_query. Aditionally on_audio_query doesn't seem to support unicode tags (#82).

**Describe the solution you'd like**
In local_music_fetcher_impl we could recurse through the library directories manually instead of using on_audio_query and get the metadata for the songs using one of the plugins mentioned in the title. This way, both of the issues mentioned above can be fixed.

**Things to consider**
Thanks to the great separation of code in mucke this should be pretty easy to do and only need changes in a few files (local_music_fetcher_impl, song_model and album_model)
flutter_audio_metadata hasn't been updated in the last year
metadata_god currently doesn't support ogg and opus (though I will contribute support for that if this change is accepted since they use rust for parsing audio files and there's a great crate with support for almost everything called [lofty-rs](https://github.com/serial-ata/lofty-rs))

**Comment by moritz-weber** [comment]
Ok, I was a bit too eager to merge this... I just realized that I get reading errors for 259 of my files (around 7%). The FfiException tells me "ID3v2: Failed to parse a frame ID".

For the most part, it's failing for whole albums - with two exceptions. I fixed one of the files by rewriting the tags with MusicBrainz Picard, so I have a broken version and a working version now.

Let me know if I should send you some files for testing.
